distribution/examples/embedding-java/src/main/java/com/predic8/membrane/api/EmbeddingJava.java

```diff
@@ -40,16 +42,20 @@ public static void main(String[] args) throws Exception {
         String path = ".*";
         int listenPort = 4000;
 
-        // The API should listen to GET requests on port 4000 with any path from any host
         ServiceProxyKey key = new ServiceProxyKey(hostname, method, path, listenPort);
+        key.setMethod(method);
```

Comment:
Is it needed to set the key and method when you are passing them in the constructor

Reply:
Can you exchange the ServiceProxyKey with an APIProxyKey? There should be one I guess.